tests/unittests/tasks/test_channels.py

```diff
@@ -0,0 +1,502 @@
+            {},
+            200,
+            [{"state": "open"}, {"state": "open"}],
+            id="assert_sum-state-simple",
+        ),
+        pytest.param(
+            "assert_sum",
+            {"from": 0, "balance_sum": 100},
+            ScenarioAssertionError,
+            'Expected sum value "100" for channel fields "balance". Actual value: "90".',
+            "GET",
+            f"http://0/api/v1/channels/0x1{1:039d}",
+            {},
+            200,
+            [{"balance": 50}, {"balance": 40}],
+            id="assert_sum-balance-mismatch",
+        ),
+        pytest.param(
+            "assert_sum",
+            {"from": 0, "state_sum": "open"},
+            ScenarioAssertionError,
+            'Expected all channels to be in "open" state.',
+            "GET",
+            f"http://0/api/v1/channels/0x1{1:039d}",
+            {},
+            200,
+            [{"state": "open"}, {"state": "closed"}],
+            id="assert_sum-state-mismatch",
+        ),
+    ],
+)
+def test_channel_task(
+    mocked_responses,
+    api_task_by_name,
+    task_name,
+    task_params,
+    expected_exception,
+    expected_exception_message,
+    expected_req_method,
+    expected_req_url,
+    expected_req_body,
+    resp_code,
+    resp_json,
+):
+    task_instance = api_task_by_name(task_name, task_params)
+
+    # If no expected method is passed we don't register an expected response
+    # Useful if an exception will be raised before the request is performed
+    if expected_req_method:
+        mocked_responses.add(
+            expected_req_method, expected_req_url, json=resp_json, status=resp_code
+        )
+
+    if expected_exception is not None:
+        with pytest.raises(expected_exception) as ex:
+            task_instance()
+        if expected_exception_message:
+            assert expected_exception_message in str(ex)
+    else:
+        response = task_instance()
+        assert response == resp_json
+
+    if expected_req_method:
+        assert mocked_responses.calls[0].request.method == expected_req_method
+        assert mocked_responses.calls[0].request.url == expected_req_url
+        assert mocked_responses.calls[0].request.body.decode() == json.dumps(expected_req_body)
+        assert mocked_responses.calls[0].response.json() == resp_json
+
+
+def test_store_channel_info(mocked_responses, dummy_scenario_runner, api_task_by_name):
+    with pytest.raises(ScenarioError) as ex:
+        api_task_by_name("store_channel_info", {"from": 0, "to": 1})
+    assert 'Required config "key" not found' in str(ex)
+
+    mocked_responses.add(
+        "GET",
+        f"http://0/api/v1/channels/0x1{1:039d}/0x2{1:039d}",
+        json={"something": 1},
+        status=200,
+    )
+    task_instance = api_task_by_name("store_channel_info", {"from": 0, "to": 1, "key": "test"})
+
+    task_instance()
+
+    assert dummy_scenario_runner.task_storage[STORAGE_KEY_CHANNEL_INFO]["test"] == {"something": 1}
```

Comment:
From reading this, I'm not entirely sure what you're testing here - adding a comment or similar would help.